tests/common.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+
+from whimsylib import actor
+from whimsylib.globals import G
```

Comment:
Now that we're doing a big refactor, I am wondering what import style is best, particularly for this global object.

```
from whimsylib.globals import G
from whimsylib.globals import G as _G
```

Both have been used throughout this and the cronenbroguelike code base. @xmunoz and @abjose, any opinions?

Reply:
I prefer the first one. The second is misleading.

Reply:
Alright just pushed a change that removes `as _G` everywhere.